unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BackpackEditorHUDV2/BackpackEditorHUDController.cs

```diff
@@ -863,14 +895,22 @@ private string GetEquipEmoteByCategory(string category)
         private string GetRandomizedName(string baseString, int limit)
         {
             currentAnimationIndexShown = (currentAnimationIndexShown + 1) % limit;
-            return baseString + (currentAnimationIndexShown + 1);
+            return $"{baseString}{(currentAnimationIndexShown + 1).ToString()}";
         }
 
         private void OnVrmExport()
         {
-            vrmExportCts?.SafeCancelAndDispose();
-            vrmExportCts = new CancellationTokenSource();
-            VrmExport(vrmExportCts.Token).Forget();
+            if (vrmWarningEnabled)
+            {
+                // WIP: Show VRM Export details panel.
+                Debug.Log("WIP: Open Details panel.");
```

Comment:
When you cant export the VRM, perhaps its better to disable the button? So this operation can be skipped

Reply:
This is a left over before implementing the details panel (not ready, not in this pr yet), sorry! The final flow to open the details panel removes this check here.
I'll see if I can commit just that other change to clean this up.